docs/en/latest/plugins/google-logging.md

```diff
@@ -0,0 +1,155 @@
+| log_id                  | Optional      | apisix.apache.org%2Flogs                                                                                                                                                                          | google logging id, refer to: [LogEntry](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry)                                                                     |
+| max_retry_count         | Optional      | 0                                                                                                                                                                                                 | max number of retries before removing from the processing pipe line                                                                                                              |
+| retry_delay             | Optional      | 1                                                                                                                                                                                                 | number of seconds the process execution should be delayed if the execution fails                                                                                                 |
+| buffer_duration         | Optional      | 60                                                                                                                                                                                                | max age in seconds of the oldest entry in a batch before the batch must be processed                                                                                             |
+| inactive_timeout        | Optional      | 10                                                                                                                                                                                                | max age in seconds when the buffer will be flushed if inactive                                                                                                                   |
+| batch_max_size          | Optional      | 100                                                                                                                                                                                               | max size of each batch                                                                                                                                                           |
+
+## How To Enable
+
+The following is an example on how to enable the `google-logging` for a specific route.
+
+### Full configuration
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "plugins": {
+        "google-logging": {
+            "auth_config":{
+                "project_id":"apisix",
+                "private_key":"-----BEGIN RSA PRIVATE KEY-----KEY-----END RSA PRIVATE KEY-----",
```

Comment:
Could we use a valid pkey?

Reply:
I think it is enough for the user to get the correct prompt in the document. The text of private_key has been updated from `KEY` to `your private key`